backend/app/adapter/sqldb/migration/30_create_progress_table.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+CREATE TABLE progress
+(
+    incident_id CHARACTER VARYING(4) NOT NULL,
+    FOREIGN KEY (incident_id) REFERENCES "incident"(id)
+        ON UPDATE CASCADE,
```

Comment:
```suggestion
        ON DELETE CASCADE ON UPDATE CASCADE,
```
If a incident is deleted, do we want to keep its progress? If not, please add `ON DELETE CASCADE`